src/main/java/com/vacation_bot/core/vacation/IVacationService.java

```diff
@@ -0,0 +1,7 @@
+package com.vacation_bot.core.vacation;
+
+public interface IVacationService {
```

Comment:
The names `VacationService` and `VacationServiceImpl` were better. Actually you can remove this interface because of `@Service` annotation usage.